style.css

```diff
@@ -99,3 +98,377 @@ section {
   font-weight: 500;
   line-height: 24px;
 }
+
+button {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 12px;
+  border: 1px solid var(--clr-purple);
+  border-radius: 8px;
+  color: var(--clr-purple);
+  background-color: transparent;
+  letter-spacing: 0.03rem;
+  font-size: 17px;
+}
+
+#Headline-Title {
+  width: 327px;
+  height: 150px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  display: flex;
+  align-items: center;
+  font-style: normal;
+}
+
+#Headline-Text {
+  width: 327px;
+  height: 146px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  font-style: normal;
+}
+
+.helpingtext {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 25px;
+  color: #7f8cff;
+  order: 0;
+  margin: 0;
+}
+
+.socialmedia ul {
+  width: 170.17px;
+  height: 20px;
+  padding: 0;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  list-style: none;
+}
+
+.headline {
+  background-color: #fff;
+  border-bottom-left-radius: 80px;
+  gap: 12px;
+  background-image: url(img/header-shapes.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding: 114px 24px 114px 24px;
+}
+
+.Grid-boxes {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto;
+  padding: 70px 25px 31px 25px;
+  row-gap: 90px;
+}
+
+.Box {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 18px;
+  gap: 14px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 18px;
+}
```

Comment:
Kindly adjust the project images so that they're displayed. You have put the image directory into `.github/workflows` but try to access them as though it is on the root. I recommend that you remove them from the `.github` directory.

![image](https://user-images.githubusercontent.com/20554583/219922407-98c6a471-644a-46ce-b0df-560626f5e752.png)